examples/fm_receiver.ts

```diff
@@ -6,7 +6,7 @@
  * (SDR - audio card) but it should be functional.
  */
 
-import { open } from '..'
+import { open } from '../lib'
```

Comment:
changing `..` to `../lib` shouldn't be necessary to run the examples, as long as you've ran `npm run build` first and you have compiled files in `dist` :thinking:

Reply:
btw! these examples are meant to be ran with `ts-node` or similar

Reply:
I see. New to the ecosystem and didn't have any build instructions :)

Reply:
Also; opening in vscode is disappointing; that guesses that the build instructions are just to run tsc, so "Start Debugging" doesn't work